recipes/sepp-refsilva128/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = "v4.3.6" %}
```

Comment:
Can you change the version to 4.3.6 and just add the "v" in the URL? The v at the beginning can much of the symantic versioning.

Reply:
good point.